### PR TITLE
Update tools_install.sh

### PR DESCRIPTION
**Pull request by hosunkang**
bash 파일 실행 시 argument 추가하여 원하는 명령 수행 가능

-m | --mscl : mscl 설치 진행
-t | --t265 : t265 설치 진행
-b | --build : 빌드 진행
-h | --help : 설명 보기

### CODE REVIEW

File: camel-tools/tools_install.sh [hunk 1]
```diff
@@ -1,56 +1,102 @@
...
+  echo "SUBSYSTEM==\"tty\", ATTRS{idVendor}==\"199b\", ATTRS{idProduct}==\"3065\", PROGRAM=\"/bin/sh -c 'echo %s{serial} | sed s/^0000__/rtk/g | sed s/^0000/main/g | sed s/^2-00/aux/g'\", SYMLINK+=\"microstrain_%c\", MODE=\"0666\"" >> 100-microstrain.rules
+  echo "SUBSYSTEM==\"tty\", ATTRS{idVendor}==\"0483\", ATTRS{idProduct}==\"5740\", PROGRAM=\"/bin/sh -c 'echo %s{serial} | sed s/^0000__/rtk/g | sed s/^0000/main/g | sed s/^2-00/aux/g'\", SYMLINK+=\"microstrain_%c\", MODE=\"0666\"" >> 100-microstrain.rules
+  sudo cp 100-microstrain.rules /etc/udev/rules.d/
+
+  cd
+  sudo udevadm control --reload-rules && sudo systemctl restart systemd-udevd && sud
+
+}
+
+install_t265(){
```
Comment (wookjinAhn) [inline camel-tools/tools_install.sh:38]: Why do you clone and install using script? what about apt-get install?

File: camel-tools/tools_install.sh [hunk 2]
```diff
@@ -1,56 +1,102 @@
...
+  sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/
+  sudo udevadm control --reload-rules && udevadm trigger
+  ./scripts/patch-realsense-ubuntu-lts.sh
+
+  BUILD_FOLDER="build"
+  if [ ! -d "$BUILD_FOLDER" ]; then
+    mkdir build
+  fi
+  cd build
+  cmake ..
```
Comment (wookjinAhn) [inline camel-tools/tools_install.sh:57]: I think this gonna be release mode.

File: camel-tools/tools_install.sh [hunk 3]
```diff
@@ -1,56 +1,102 @@
...
+  if [ ! -d "$BUILD_FOLDER" ]; then
+    mkdir build
+  fi
+  cd build
+  cmake .. -DCMAKE_BUILD_TYPE=Release
+  make
+  sudo make install
+}
+
+VALID_ARGS=$(getopt -o mtbh --long mscl,t265,build,help -- "$@")
```
Comment (wookjinAhn) [inline camel-tools/tools_install.sh:72]: This was what I wanted to do. Awesome!

**Review by wookjinAhn — DISMISSED** [review]
I have a question about installing librealsense and found mistake in the cmake process.

File: camel-tools/tools_install.sh (hunk 2 above)
Comment (hosunkang) [reply camel-tools/tools_install.sh]: OMG... Why this is not changed. I will do it.

File: camel-tools/tools_install.sh (hunk 1 above)
Comment (hosunkang) [reply camel-tools/tools_install.sh:38]: Do you mean t265 library can be installed by using apt-get? I didn't know that. Tell me how to do.

**Comment by ohDDo** [comment]
Awesome!

File: camel-tools/tools_install.sh (hunk 1 above)
Comment (wookjinAhn) [reply camel-tools/tools_install.sh:38]: https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md#installing-the-packages

This is the way how to install

**Review by wookjinAhn — COMMENTED** [review]
good

**Review by wookjinAhn — APPROVED** [review]
good

**Pull request merged**